src/cryptonote_core/tx_blink.h

```diff
@@ -181,6 +181,10 @@ class blink_tx {
      */
     void fill_serialization_data(crypto::hash &tx_hash, uint64_t &height, std::vector<uint8_t> &quorum, std::vector<uint8_t> &position, std::vector<crypto::signature> &signature) const;
 
+    /// Wrapper around the above that can be called with a serializable_blink_metadata
+    template <typename T>
```

Comment:
Yes, because we type we actually need here is defined in cryptonote_protocol, which isn't included here to keep cryptonote_core free of a dependency on cryptonote_protocol.  (Also see the comment just before the previous function that mentions this).